### PR TITLE
Rename native contracts

### PR DESCRIPTION
**Pull request by erikzhang**
1. Rename `ManagementContract` to `ContractManagement`.
2. Rename `DesignationContract` to `RoleManagement`.

### CODE REVIEW

**Comment by Tommo-L** [comment]
BTW, rename `Committee` to `Council`

**Comment by erikzhang** [comment]
> BTW, rename Committee to Council

It's another pr.

**Comment by devhawk** [comment]
Should We also change NeoToken/GasToken back to NEO and GAS?

**Comment by cloud8little** [comment]
> Should We also change NeoToken/GasToken back to NEO and GAS?

Agree

**Comment by erikzhang** [comment]
> Should We also change NeoToken/GasToken back to NEO and GAS?

I don't think so. `NEO` is ambiguous, does it mean token or blockchain? And it has the same name as the root namespace, which will cause many conflicts.

**Comment by superboyiii** [comment]
Do you think it's need to be merged into Preview4? @erikzhang

**Comment by Tommo-L** [comment]
UT failed.

File: src/neo/SmartContract/Native/RoleDesignation.cs [hunk 1]
```diff
@@ -14,12 +12,12 @@
 
 namespace Neo.SmartContract.Native
 {
-    public sealed class DesignationContract : NativeContract
+    public sealed class RoleDesignation : NativeContract
```
Comment (vncoelho) [inline src/neo/SmartContract/Native/RoleDesignation.cs:15]: Maybe keep with `ContractRoleDesignation` to follow `ContractManagement`

Comment (erikzhang) [reply src/neo/SmartContract/Native/RoleDesignation.cs]: I don't understand.

Comment (erikzhang) [reply src/neo/SmartContract/Native/RoleDesignation.cs]: Now it is `RoleManagement`.

**Review by shargon — DISMISSED** [review]
It's ok for me, but I prefer to merge this after preview4

**Comment by devhawk** [comment]
> It's ok for me, but I prefer to merge this after preview4

I agree 100%

**Comment by erikzhang** [comment]
Now it can be merged.

**Review by shargon — APPROVED** [review]

**Pull request merged**